cunumeric/array.py

```diff
@@ -3048,6 +3048,54 @@ def setflags(self, write=None, align=None, uic=None):
         """
         self.__array__().setflags(write=write, align=align, uic=uic)
 
+    @add_boilerplate()
+    def searchsorted(
+        self: ndarray,
+        v: Union[int, float, ndarray],
+        side: str = "left",
+        sorter: Optional[ndarray] = None,
+    ) -> Union[int, ndarray]:
+        """a.searchsorted(v, side='left', sorter=None)
+
+        Find the indices into a sorted array a such that, if the corresponding
+        elements in v were inserted before the indices, the order of a would be
+        preserved.
+
+        Parameters
+        ----------
+        v : scalar or array_like
+            Values to insert into a.
+        side : ``{'left', 'right'}``, optional
+            If 'left', the index of the first suitable location found is given.
+            If 'right', return the last such index. If there is no suitable
+            index, return either 0 or N (where N is the length of a).
+        sorter : 1-D array_like, optional
+            Optional array of integer indices that sort array a into ascending
+            order. They are typically the result of argsort.
+
+        Returns
+        -------
+        indices : int or array_like[int]
+            Array of insertion points with the same shape as v, or an integer
+            if v is a scalar.
+
+        Availability
+        --------
+        Multiple GPUs, Multiple CPUs
+        """
+
```

Comment:
I think you might want to check that self is a 1D array somewhere here, if not, return the error.
I assume we would also want to check that V is a 1D array or scalar here as well

Reply:
Actually V can be a multidimensional array. This example works just fine:
```
arr = np.array([1, 2, 3, 4, 5], dtype=int)
arr2= np.array([[1.0, 4.1],[0.0, 3.2]], dtype = float)
res = np.searchsorted(arr, arr2)
```

Reply:
I added a check for self and sorter (if available).